apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/IterableContainsInOrderAtLeast1ElementsOfAssertionSpec.kt

```diff
@@ -0,0 +1,37 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import org.spekframework.spek2.Spek
+import kotlin.reflect.KFunction3
+
+class IterableContainsInOrderAtLeast1ElementsOfAssertionSpec : Spek({
+    include(BuilderSpec)
+}) {
+    object BuilderSpec : ch.tutteli.atrium.specs.integration.IterableContainsInOrderAtLeast1ValuesAssertionsSpec  (
```

Comment:
Note you are not implementing an `inOrder.atLeast(1)` behaviour but `inOrder.only` thus use the following (the one you wrote does not exist)
```suggestion
    object BuilderSpec : ch.tutteli.atrium.specs.integration.IterableContainsInOrderOnlyEntriesAssertionsSpec  (
```